app/kuma-dp/pkg/dataplane/metrics/profiles.go

```diff
@@ -152,17 +153,47 @@ var basicProfile = []selectorFunction{
 	// end of dashboards
 }
 
+var basicProfileLabels = []selectorFunction{
+	selectorToFilterFunction(v1alpha1.Selector{
+		Type:  v1alpha1.ExactSelectorType,
+		Match: names.GetAdsClusterName(),
+	}),
+	selectorToFilterFunction(v1alpha1.Selector{
+		Type:  v1alpha1.ExactSelectorType,
+		Match: names.GetAccessLogSinkClusterName(),
+	}),
+	selectorToFilterFunction(v1alpha1.Selector{
+		Type:  v1alpha1.ExactSelectorType,
+		Match: names.GetEnvoyAdminClusterName(),
+	}),
+	selectorToFilterFunction(v1alpha1.Selector{
+		Type:  v1alpha1.ExactSelectorType,
+		Match: names.GetMetricsHijackerClusterName(),
+	}),
+	selectorToFilterFunction(v1alpha1.Selector{
+		Type:  v1alpha1.PrefixSelectorType,
+		Match: names.GetOpenTelemetryClusterPrefix(),
+	}),
+	selectorToFilterFunction(v1alpha1.Selector{
+		Type:  v1alpha1.PrefixSelectorType,
+		Match: names.GetTracingClusterPrefix(),
+	}),
```

Comment:
Should we already add all clusters starting with `_` for future proofing with https://github.com/kumahq/kuma/issues/8469 ?